Chart.yaml

```diff
@@ -16,8 +16,15 @@ maintainers:
   - name: alexnuttinck
     email: alexandre.nuttinck@cetic.be
     url: https://github.com/alexnuttinck
+  - name: daniel-yavorovich
```

Comment:
For the moment, I can't add you as a maintainer of the Chart, you will be maintainer of the nifi-registry Chart :) I can consider to add you in the case you participate in bug fixes, feature implementations...

Reply:
sure, I have removed it. Thank you for explain!